native/cocos/editor-support/spine-wasm/spine-skeleton-instance.h

```diff
@@ -83,6 +79,7 @@ class SpineSkeletonInstance {
     uint32_t _disposeListenerID = 0;
     uint32_t _completeListenerID = 0;
     uint32_t _eventListenerID = 0;
+    uint32_t _trackListenerID = 0;
```

Comment:
t is better to use a more meaningful name: _trackListenerID -> _trackEntryListenerID